tools/PublishProject/Program.cs

```diff
@@ -80,69 +83,80 @@ public static int Main(string[] args)
                                     string rid,
                                     string framework)
         {
-            var runtimeString = Regex.Replace(rid, "(\\d|\\.)*-", "-");
+            var runtimeString = Regex.Replace(rid, @"(\d|\.)*-", "-");
+            var buildIdentifier = $"{runtimeString}-{framework}";
 
-            // Disable for now, while travis isn't working correctly.
-            if (runtimeString.Contains("osx-")) {
-                return;
+            // Rename framework name to mono for non-windows platforms
+            if (runtimeString.Contains("ubuntu-") || runtimeString.Contains("osx-"))
+            {
+                buildIdentifier.Replace("dnx451", "mono");
             }
             
-            // Simplify Ubuntu to Linux
-            runtimeString = runtimeString.Replace("ubuntu", "linux");
```

Comment:
@hal-ler @david-driscoll what's the original intention here? Ubuntu != Linux since the implementation packages are different among distros.